App.tsx

```diff
@@ -15,6 +15,7 @@ const routes = {
   Alternate: "Alternate",
   Test: "Test",
   First: "First",
+  List: "List"
```

Comment:
This is a component, do'n create a route for this.

Reply:
removed it